config.json

```diff
@@ -1,37 +1,3 @@
 {
-    "website_name": "Binary.com",
-    "display_languages": "EN,ID,RU,ES,FR,PT,DE,ZH_CN,PL,AR,ZH_TW,VI,IT",
-    "static": {
-        "url": "https://static.binary.com/",
-        "version": "3.8.0"
-    },
-    "marketing": {
-        "google_tag_manager": "GTM-MZWFF7"
-    },
-    "customer_support": {
-        "email": "helpdesk@binary.com",
-        "telephone": {
-            "Canada": "+1 (450) 823 1002",
-            "Australia": "+61 (02) 8294 5448",
-            "Ireland": "+353 (0) 76 888 7500",
-            "Poland": "+48 58 881 00 02",
-            "United Kingdom": "+44 (0) 1666 800042",
-            "Germany": "+49 0221 98259000"
-        },
-        "tollfree_number": {
-            "Australia": "1800 093570",
-            "Indonesia": "0018030113641",
-            "Ireland": "1800931084",
-            "Russia": "8 10 8002 8553011",
-            "United Kingdom": "0800 011 9847"
-        },
-        "default_telephone": {
-            "country": "United Kingdom"
-        },
-        "skype_helpline": {
-            "EN": "binaryhelpline",
-            "ID": "binaryhelplineid",
-            "RU": "binaryhelplineru"
-        }
-    }
+    "website_name": "Binary.com"
```

Comment:
this was kept so that config.json is not empty, i am considering other stated in other pull request